code/modules/status_system/statusEffects.dm

```diff
@@ -659,6 +659,9 @@
 				var/mob/living/carbon/human/H = owner
 				prot = (1 - (H.get_heat_protection() / 100))
 
+			if (H.traitHolder && H.traitHolder.hasTrait("burning")) //trait 'burning' is human torch
```

Comment:
```suggestion
			if (H.traitHolder?.hasTrait("burning")) //trait 'burning' is human torch
```
Slightly more concise- `?.` is sort of like "if the thing on the left exists (isn't null), then continue; otherwise don't"